modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/util/ServiceContextUtil.java

```diff
@@ -22,12 +23,14 @@
 public class ServiceContextUtil {
 
 	public static ServiceContext createServiceContext(
-		Locale locale, ObjectEntry objectEntry, long userId) {
+		Locale locale, ObjectEntry objectEntry, long userId,
+		ModelPermissions modelPermissions) {
```

Comment:
Is this order correct?
What was the pattern that you followed?

Reply:
@marcelabc I updated the code